core/capabilities/remote/types/remote_messages.proto

```diff
@@ -48,3 +48,10 @@ message TriggerEventMetadata {
   string trigger_event_id = 1;
```

Comment:
I'm not sure why, but suddenly leaving this name as is was causing panics due to a file called message.proto already existing in the global proto registry. See: https://protobuf.dev/reference/go/faq/#namespace-conflict

I spent a bit of time trying to debug, but ended up just renaming it.

Reply:
Where was the panic? I didn't see any problems before.. would prefer not renaming if possible.
Maybe the problem is in go:generate being relative vs absolute? Have you tried having a  top-level one that covers multiple proto files?

Reply:
I'll take another look

Reply:
The panic was due to a file naming conflict with wsrpc/message.proto. My best guess is that we're now importing these protos at some point which is causing the duplicate entry in proto's internal GRPC message registry.

Reply:
OK... I narrowed this down to the command we were using to generate the proto file in types.go.

I used a not so pretty modification which ensures the file descriptor it generates for the proto registry won't clash with other files.

I suspect that what happened is the original proto file was generated from the root of chainlink (but I couldn't find the command that was used anywhere 🤷 ), thus ensuring we'd have that fully qualified path. The types.go command didn't generate it from there, and generated it from the package, stripping the fully qualified path.